about.html

```diff
@@ -0,0 +1,120 @@
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+			rel="stylesheet"
+		/>
+		<link
+			rel="stylesheet"
+			href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+		/>
+		<title>About</title>
+	</head>
+	<body>
+		<header id="header">
+			<div class="top-banner">
+				<div class="banner">
+					<img src="./images/Facebook.svg" alt="" />
+					<img src="./images/twitter.svg" alt="" />
+					<p>Logout</p>
+					<p>My Page</p>
+					<p>English</p>
+				</div>
+			</div>
+			<div class="menu" id="mobile-menu">
+				<div class="menu-bottom">
+					<a href="./index.html"
+						><img src="./images/logo.svg" alt=""
+					/></a>
+					<div class="left-menu">
+						<ul class="left">
+							<li><a href="./index.html">Home</a></li>
+							<li><a class="active" href="#">About</a></li>
+							<li><a href="">Program</a></li>
+							<li><a href="">News</a></li>
+						</ul>
+						<button>CC compaign</button>
+					</div>
+				</div>
+			</div>
+			<div id="humburger">
+				<span class="material-symbols-outlined"> menu </span>
+			</div>
+		</header>
+		<section id="aboutHeadline">
+			<div class="hero-content">
+				<div class="title">
+					<h1>2021 Glocal Mission Summit</h1>
+					<hr />
+				</div>
+				<div class="about">
+					<p>
+						The Glocal Mission Summit is a biannual event where
+						people engage in conversations around mission and
+						imagine innovative ways to engage and embrace our local
+						communities that are becoming increasingly diverse.
+					</p>
+				</div>
+				<div class="contact">
+					<p>
+						Please contact us per Email for nay further questions
+						about CC Global Summit 2015!
+					</p>
+					<h4>ccsummit2015@cckorea.org</h4>
+				</div>
+			</div>
+		</section>
+		<section id="logoSection">
+			<div class="logo">
+				<h1>How do we join God's mission locally and globally?</h1>
+				<hr />
+				<p>
+					Join us for this six-part Glocal pilot podcast as we meet
+					ministry leaders from around North America as they live out
+					the gospel in their own contexts. Stories of God at work -
+					globally and locally - abound. Stories of church planters,
+					missionaries, community developers, evangelists, campus
+					pastors, and many more continue to inspire lay leaders,
+					pastors, and para-church people to re-engage, re-imagine,
+					and rejoice in joining God on mission in their own contexts.
+				</p>
+				<img src="/scss/images/resonate-footer-crcna-logo.png" alt="" />
+			</div>
+		</section>
+		<section id="LastYear">
+			<div class="last">
+				<div class="content">
+					<h1>Episodes</h1>
```

Comment:
- [ ] Just as i suggested above, Following the best practice, you are meant to have only 1 `h1` tag in a page and that should be the main title, I'll suggest you update this to `h2 or h3` :+1: :smile: